test/exec.js

```diff
@@ -1,9 +1,11 @@
-const { execSync } = require('child_process');
-const exec = (cmd) => {
+const {execSync} = require('child_process');
+const {FIXTURE} = require('./fixtures');
+
+const exec = (arg) => {
   try {
-    return execSync(
-      'node src/' + cmd.split(' ')[0] + '.js' + cmd.substring(cmd.indexOf(' '))
-    ).toString();
+    const command = arg.split(' ')[0];
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/codingstyle-cn/hlj/pullRequest?prid=7143429)
```suggestion
    const command = arg.split(" ")[0];
```